src/App.js

```diff
@@ -1,13 +1,38 @@
 import './App.css';
-
+import { Link, Route, Routes } from 'react-router-dom';
 import React from 'react';
 
 import MainCalculator from './components/calculator';
+import Home from './components/home';
+import Quote from './components/quote';
 
 class App extends React.PureComponent {
   render() {
     return (
-      <MainCalculator />
+      <>
+        <header>
+          <nav>
+            <ul>
+              <li>
+                <Link to="/home">Home</Link>
+              </li>
+              <li>
+                <Link to="/calculator">Calculator</Link>
+              </li>
+              <li>
+                <Link to="/quote">Quotes</Link>
+              </li>
+            </ul>
+          </nav>
+          <h1>Math Magicians</h1>
+        </header>
+        <Routes>
+          <Route path="/Math-magicians" element={<Home />} />
+          <Route path="/calculator" element={<MainCalculator />} />
+          <Route path="/home" element={<Home />} />
```

Comment:
- [ ] Why do you have one component with two different route? I think it would be better if you use the home component as the index route. `<Route path="/" element={<Home />} />`